src/utils/getGuildMemberDetails.ts

```diff
@@ -0,0 +1,27 @@
+import { INTERNAL_SERVER_ERROR } from "../constants/responses";
+import { DISCORD_BASE_URL } from "../constants/urls";
+import { env } from "../typeDefinitions/default.types";
+import { discordMemberDetails } from "../typeDefinitions/discordMessage.types";
+
+export async function getGuildMemberDetails(
+  userId: string,
+  env: env
+): Promise<discordMemberDetails | string> {
+  const getGuildMemberDetailsUrl = `${DISCORD_BASE_URL}/guilds/${env.DISCORD_GUILD_ID}/members/${userId}`;
```

Comment:
Nit: can we avoid adding a discord guild id, since it is a constant, we can create this URL in some constant file